content/about/cloudify-manager.md

```diff
@@ -1,18 +1,18 @@
 ---
 layout: bt_wiki
-title: Cloudify Manager
+title: Cloudify Management Console
```

Comment:
Shouldn't we keep the title? Page looks like describing Cloudify Manager in general, not Cloudify Management Console (aka Stage).

Reply:
fixed